spec/fixtures/workspace/ab/123/cd/4567/content_metadata.xml

```diff
@@ -4,14 +4,14 @@
       <attr name="pageNumber">3</attr>
       <attr name="pageLabel">ii</attr>
       <attr name="googlePageTag">IMAGE_ON_PAGE,IMPLICIT_PAGE_NUMBER</attr>
-      <file id="00000001.jp2" format="JPEG2000" mimetype="image/jp2" size="169627" shelve="no" deliver="no" preserve="yes">
+      <file id="00000001.jp2" format="JPEG2000" mimetype="image/jp2" size="169627" shelve="no" publish="yes" preserve="yes">
```

Comment:
Changes to this file were made to update what the content metadata should look like.